core/chains/evm/txm/clientwrappers/dual_broadcast_client.go

```diff
@@ -0,0 +1,130 @@
+package clientwrappers
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math/big"
+	"net/http"
+	"net/url"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/txm/types"
+)
+
+type DualBroadcastClientKeystore interface {
+	SignMessage(ctx context.Context, address common.Address, message []byte) ([]byte, error)
+}
+
+type DualBroadcastClient struct {
```

Comment:
It appears to me, this DualBroadcast client is for the OEV work.
Is there more info on this feature? 
Hard to understand the code without knowing context on this.
Also, it seems worrying to me that we are writing a new raw client here, and not trying to reuse our MultiNode/Node code. Even if this is a separate single RPC, why not wrap the Node around this?

Reply:
There is a lot to unpack here. The problem is that the custom URL is public and unreliable in terms of responses (perhaps it doesn't even support all calls), so it doesn't seem optimal to create a new multinode instance for it. We'll not be getting accurate indications, and we still only have a single RPC endpoint. To make matters worse, multinode is RPC intensive and I don't want us to be rate-limited at any point for no reason.
We already have custom TXM clients for the existing TXM to wrap some necessary behavior, I think this is no different than that. Let me know if there are any benefits I'm missing.